code/src/cljs/sixsq/nuvla/ui/edges/events.cljs

```diff
@@ -367,12 +368,14 @@
 (reg-event-fx
   ::get-nuvlabox-releases
   (fn [{:keys [db]} _]
-    {:db                  (assoc db ::spec/nuvlabox-releases nil)
-     ::cimi-api-fx/search [:nuvlabox-release
-                           {:select  "id, release, pre-release, release-notes, url, compose-files"
-                            :orderby "release-date:desc"
-                            :last    10000}
-                           #(dispatch [::set-nuvlabox-releases %])]}))
+    (let [view-data (get-in db [::session-spec/session :acl :view-data])]
```

Comment:
active claim is retrieved from ::session-spec/session with following statement in different places in UI:
`(or (:active-claim session) (:user session))`

An enhancement is to store active-claim in db when we set session to avoid duplicating this statement.

Reply:
I think I misunderstood something fundamentally here :)

Reply:
> An enhancement is to store active-claim in db when we set session to avoid duplicating this statement

or a write a function 💡 😄